admin_guide/sdn_troubleshooting.adoc

```diff
@@ -395,13 +394,6 @@ Of course the opposite traffic flows are also possible.
 
 Check that `sysctl net.ipv4.ip_forward` is set to 1 (and check the host if this is a VM)
 
-==== Is `firewalld` Disabled?
-
-Check that `firewalld` is disabled using `systemctl status firewalld`.
-If it is running, you either need to disable it, or check
-that it is not blocking traffic.  That is outside the scope of this
-guide.
```

Comment:
Actually, after helping someone deal with another firewalld-related issue, I realize we should keep this text for now; although we are compatible with firewalld in the general case, we are not compatible with its default configuration (which will block incoming connections that OpenShift wants). So for now the text is basically correct: if it is running, you need to make sure that it is not blocking traffic, and that is outside the scope of this guide.